feature/dynamodb/expression/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/aws/aws-sdk-go-v2/feature/dynamodb/expression
+
+go 1.15
+
+require (
+	github.com/aws/aws-sdk-go-v2 v0.30.0
+	github.com/aws/aws-sdk-go-v2/feature/dynamodb/attributevalue v0.0.0-00010101000000-000000000000
```

Comment:
this could be updated to be the gohash of the SDK now that attributevalue util is merged in.